lib/rubocop/cop/style/stabby_lambda_parentheses.rb

```diff
@@ -0,0 +1,92 @@
+# encoding: utf-8
+module RuboCop
+  module Cop
+    module Style
+      # Check for parentheses around stabby lambda arguments
+      class StabbyLambdaParentheses < Cop
+        include ConfigurableEnforcedStyle
+
+        def on_send(node)
+          return unless valid_node?(node)
+
+          if style == :require_parentheses
+            if !parentheses?(node)
+              missing_parentheses(node)
+            else
+              correct_style_detected
+            end
+          elsif parentheses?(node)
+            unwanted_parentheses(node)
+          else
+            correct_style_detected
+          end
+        end
+
+        def autocorrect(node)
+          if style == :require_parentheses
+            missing_parentheses_corrector(node)
+          elsif style == :require_no_parentheses
+            unwanted_parentheses_corrector(node)
+          end
+        end
+
+        def missing_parentheses(node)
+          add_offense(node_args(node), :expression,
+                      'Use with parentheses when there are parameters.') do
```

Comment:
Extract this message to a constant. I'd also rephrase it to something like `Wrap stabby lambda arguments with parentheses.`.